test/Confluent.SchemaRegistry.IntegrationTests/Tests/proto/compile_linux.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
set -e

Reply:
Might want to consider using a makefile instead.
It is a more intuitive experience and less files to maintain.

```
_UNAME_S := $(shell uname -s)
ifeq ($(_UNAME_S),Darwin)
_PLATPATH := macosx_x64
else
_PLATPATH := linux_x64
endif

all:
     ~/.nuget/packages/grpc.tools.1.16.0/tools/$(_PLATPATH)/protoc \
          ...
```

Reply:
considered